containers/Ideas/IdeaCreate.js

```diff
@@ -164,16 +187,17 @@ export const IdeaCreate = props => {
           ...(editing
             ? {
               name: idea.name,
-              description: idea.description ,
+              description: idea.description,
             }
             : {
               name: '',
-              description: '' ,
+              description: '',
+
             }),
 
           labels: [],
           category: selectedCategory,
-          imageCopyrightChecked: imageChecked
+          imageCopyrightChecked: true
```

Comment:
Oh, I only changed this to test the post. We should change it back before we merge.